sitemap_utils.py

```
@@ -0,0 +1,367 @@
+            
+            logger.info(f"Parsed sitemap with {len(sitemap_urls)} URLs")
+            return sitemap_urls, []
+            
+        except ParseError as e:
+            logger.error(f"XML parsing error: {e}")
+            return [], []
+        except Exception as e:
+            logger.error(f"Error parsing sitemap XML: {e}")
+            return [], []
+
+    def _process_sitemap(self, sitemap_url: str) -> List[SitemapURL]:
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)